lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_firebase_test.rb

```diff
@@ -0,0 +1,163 @@
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :test_apk_path,
+            description: 'The test APK',
+            type: String,
+            verify_block: proc do |value|
+              UI.user_error!("Invalid test APK: #{value}") unless File.exist?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :model,
+            description: 'The device model to run',
+            type: String,
+            verify_block: proc do |value|
+              model_names = Fastlane::FirebaseDevice.valid_model_names
+              UI.user_error!("Invalid Model Name: #{value}. Valid Model Names: #{model_names}") unless model_names.include?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :version,
+            description: 'The device version to run',
+            type: Integer,
+            verify_block: proc do |value|
+              version_numbers = Fastlane::FirebaseDevice.valid_version_numbers
+              UI.user_error!("Invalid Version Number: #{value}. Valid Verison Numbers: #{version_numbers}") unless version_numbers.include?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :locale,
+            description: 'The locale code to run in',
+            type: String,
+            default_value: 'en',
+            verify_block: proc do |value|
+              locale_codes = Fastlane::FirebaseDevice.valid_locales
+              UI.user_error!("Invalid Locale: #{value}. Valid Locales: #{locale_codes}") unless locale_codes.include?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :orientation,
+            description: 'Which orientation to run the device in',
+            type: String,
+            default_value: 'portrait',
+            verify_block: proc do |value|
+              orientations = Fastlane::FirebaseDevice.valid_orientations
+              UI.user_error!("Invalid Orientation: #{value}. Valid Orientations: #{orientations}") unless orientations.include?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :type,
+            description: 'Which type of test are we running?',
+            type: String,
+            default_value: 'instrumentation',
+            verify_block: proc do |value|
+              types = Fastlane::FirebaseTestRunner::VALID_TEST_TYPES
+              UI.user_error!("Invalid Test Type: #{value}. Valid Types: #{types}") unless types.include?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :test_run_id,
+            description: 'A unique ID used to identify this test run',
+            type: String
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :results_output_dir,
+            description: 'Where should we store the results of this test run?',
+            type: String
+          ),
```

Comment:
You should still keep the `default_value_dynamic: true` for both of those imho, to indicate in the action's generated documentation that if the user does not provide a default value, one will be dynamically derived/generated for them

Reply:
Oh interesting – the examples that I was able to find mostly only had it with `default_value`, but turns out there were examples that had it! Re-added in https://github.com/wordpress-mobile/release-toolkit/pull/355/commits/1c1a8bccfc62ecdd833813950f05253b69bf0715

Reply:
> All that is unlike your case of a UUID default value, which would change on every, single, run.

Yeah this makes sense for sure! I'm good with how it is now 🙂